lib/tasks/install_secrets.rake

```diff
@@ -45,26 +45,40 @@ task :install_secrets, [] do
     end
   end
 
-  database_secrets = secrets.delete('database')
+  database_secrets = secrets.delete(:database)
+  write_yaml_file("config/database.yml", {
+    production: {
+      database: database_secrets[:name],
+      host: database_secrets[:url],
+      port: database_secrets[:port],
+      username: database_secrets[:username],
+      password: database_secrets[:password],
+      adapter: "postgresql",
+      encoding: "unicode",
+      pool: '<%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>'
+    }
+  })
 
-  File.open(File.expand_path("config/database.yml"), "w") do |file|
-    file.write({
-      'production' => {
-        'database' => database_secrets['name'],
-        'host' => database_secrets['url'],
-        'port' => database_secrets['port'],
-        'username' => database_secrets['username'],
-        'password' => database_secrets['password'],
-        'adapter' => "postgresql",
-        'encoding' => "unicode",
-        "pool" => '<%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>'
-      }
-    }.to_yaml[4..-1])
-  end
+  scout_secrets = secrets.delete('scout')
+  write_yaml_file("config/scout_apm.yml", {
+    production: {
+      key: scout_secrets[:license_key],
+      name: "highlights (#{env_name})",
+      # crude way to disable scout by environment
+      monitor: !/noscout/.match?(env_name),
+      ignore: %w(/ping)
+    }
+  })
+
+  write_yaml_file("config/secrets.yml", {
+    production: secrets
+  })
+end
 
-  File.open(File.expand_path("config/secrets.yml"), "w") do |file|
-    # write the secrets hash as yaml, getting rid of the "---\n" at the front
-    file.write({'production' => secrets}.to_yaml[4..-1])
+def write_yaml_file(filename, hash)
+  File.open(File.expand_path(filename), "w") do |file|
+    # write the hash as yaml, getting rid of the "---\n" at the front
+    file.write(hash.deep_stringify_keys.to_yaml[4..-1])
```

Comment:
would something like `yaml.gsub("---\n", '')` be better than these 2 lines?  it's one line and self documenting?

Reply:
good point.  we should do that once we move it somewhere

Reply:
delayed to https://app.zenhub.com/workspaces/openstax-unified-5b71aabe3815ff014b102258/issues/openstax/unified/673